tools/cli/commands-packages.js

```diff
@@ -1686,11 +1705,13 @@ main.registerCommand({
 
       // If the user explicitly mentioned some packages to upgrade, they must
       // actually end up in our solution!
-      _.each(options.args, function (packageName) {
-        if (! projectContext.packageMap.getInfo(packageName)) {
-          buildmessage.error(packageName + ': package is not in the project');
-        }
-      });
+      if (options.args.length !== 0) {
```

Comment:
Is this `if` statement even necessary?  Doesn't `upgradePackageNames` _have_ to be an array at this point?  If so, `each` will just do nothing anyway.  It just seems redundant.